server/files.go

```diff
@@ -170,6 +149,30 @@ func decodeCreateFileRequest(_ context.Context, request *http.Request) (interfac
 		}
 	}
 
+	bs, err := ioutil.ReadAll(request.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	h := strings.ToLower(request.Header.Get("Content-Type"))
+	if strings.Contains(h, "application/json") {
+		// Read body as ACH file in JSON
+		f, err := ach.FileFromJSON(bs)
+		if f != nil {
+			req.File = f
+		}
+		req.parseError = err
+	} else {
+		// Attempt parsing body as an ACH File
+		r = bytes.NewReader(bs)
+		achReader := ach.NewReader(r)
+		achReader.SetValidation(req.validateOpts)
+	
+		f, err := achReader.Read()
+		req.File = &f
+		req.parseError = err
+	}
+
```

Comment:
All of this code is unchanged, except the line where we set the validation options in the `ach.NewReader`: `achReader.SetValidation(req.validateOpts)`